awkward/array/jagged.py

```diff
@@ -406,6 +407,56 @@ def __array_ufunc__(self, ufunc, method, *inputs, **kwargs):
             return None
         else:
             return JaggedArray(starts, stops, result)
+        
+
+    def argproduct(self, other):
+        import awkward.array.table 
+        if not isinstance(other, JaggedArray):
+            raise ValueError("array given isn't instance of JaggedArray; need JaggedArrays to proceed")
+        
+        if (len(self._starts) != len(other)):
+            raise ValueError("Number of events in each array must be equal")
+        
+        starts1 = self._starts
+        stops1 = self._stops
+        counts1 = stops1 - starts1
+
+        starts2 = other.starts
+        stops2 = other.stops
+        counts2 = stops2 - starts2
+        
```

Comment:
should we check that counts1.shape == counts2.shape?

Reply:
That won't be necessary, as we are doing shape checking for starts array. The shape of counts and starts are same.